common/djangoapps/util/keyword_substitution.py

```diff
@@ -17,7 +17,7 @@
     above other modules in the dependency tree and acts like a global var.
     Then we can call substitute_keywords_with_data where substitution is
     needed. Currently called in:
-        - LMS: Bulk emails and emails on enrollment
+        - LMS: Bulk emails, emails on enrollment, and course announcements
```

Comment:
Seeing as we appear to be regularly updating this list, would it make sense to break things out into sublists?

```
Currently called in:
- LMS:
    - Bulk emails
    - emails on enrollment
    - course announcements
- CMS
    - Test emails on enrollment
```